pages/common/figlet.md

```diff
@@ -0,0 +1,15 @@
+# figlet
+
+> Display large characters made up of ordinary screen characters.
+
+- Use default settings:
```

Comment:
We generally use descriptions which are action oriented. Like describing what the command does.

In this case, how about - `Generate an ascii banner for "Hello World"` ?